README.md

```diff
@@ -38,13 +38,14 @@ We will explain how to develop this application with Scalar DB and what differs
 
 This document contains different underlying storage/database (such as Cassandra, Cosmos DB) set up and configurations, Please follow any one of them.
 
-## Install prerequisites
+## Install prerequisites and Configure Scalar DB
 
-Following softwares are required to start the demonstration.
+Following softwares and Scalar DB connection configuration are required to start the demonstration.
 
-* Install [Scalar DB on Cassandra prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cassandra.md#install-prerequisites) if you use Cassandra as a backend database.
-* Install [Scalar DB on Cosmos DB prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cosmosdb.md#install-prerequisites) if you use Cosmos DB as a backend database.
-* Install [Scalar DB on DynamoDB prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-dynamodb.md#install-prerequisites) if you use DynamoDB as a backend database.
+* **Install prerequisites** and **Configure Scalar DB** configuration ([**scalardb.properties**](backend/QA/src/main/resources/scalardb.properties)) file based on backend database. 
```

Comment:
```suggestion
* Install prerequisites and configure Scalar DB ([**scalardb.properties**](backend/QA/src/main/resources/scalardb.properties)) file on the basis of backend database. 
```

Reply:
Thank you for your suggestion.
Suggestion integrated.